elastic/kibana
# [Stateful sidenav] Enable on-prem

In this PR I've enabled the new space solution view (which renders the different solution side navigations) for on-prem users.

# How to test

* Make sure you don't have any yml setting that were previously used to activate the solution view

```yml
### Both those setting must **not** exist or be commented out ###

# xpack.spaces.experimental.forceSolutionVisibility
# xpack.cloud.id
```

* Start Kibana
* Edit the current space
* You should see the solution view drop down

# Release notes
The new space solution view is now available for on-prem users. The space solution views allows solution centric navigation similar to our serverless offering. 

Fixes https://github.com/elastic/kibana/issues/203107
